js/chunk.js

```
@@ -47,9 +47,11 @@ export class Chunk {
 			// Initialize base terrain
 			for (let x = 0; x < this.size; x++) {
 				for (let z = 0; z < this.size; z++) {
-					// Console log memory usage
-					const memoryUsage = window.performance.memory.usedJSHeapSize / (1024 * 1024);
-					console.log(`Memory usage: ${memoryUsage.toFixed(2)} MB of ${window.performance.memory.jsHeapSizeLimit / (1024 * 1024)} MB`);
+					// Conditionally log memory usage if available in the browser
+					if (window.performance && window.performance.memory) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)